requirements.txt

```diff
@@ -0,0 +1,10 @@
+numpy>=1.14.5
+scipy>=1.1.0
+hankel>=0.3.6
+emcee
+pyevtk
+pyvista
+six
+matplotlib
+pytest
+pytest-cov
```

Comment:
Please remove pyvista, matplotlib, pytest and pytest-cov from this list.

Reply:
Because of this, the py34 build fails and we don't need to install pyvista and matplotlib on travis.